### ISSUE TITLE
SPRING_CLOUD_APPLICATION_GROUP not set

### ISSUE THREAD

**Issue by ericbottard** [spring-cloud/spring-cloud-deployer-mesos#47]
A quick github search for SPRING_CLOUD_APPLICATION_GROUP yields nothing in this repo, which may be a problem.

This is the SO thread that triggered this search: http://stackoverflow.com/questions/42225015/programmatically-getting-the-stream-name

**Comment by ilayaperumalg** [comment]
Is this supposed to be in https://github.com/spring-cloud/spring-cloud-deployer-cloudfoundry ?

**Comment by ericbottard** [comment]
no, CF and k8s have it (haven't checked YARN). Mesos seems to be missing it

**Comment by trisberg** [comment]
This was part of the 1.1.x versions but Mesos lags behind and is currently released at 1.0.4 and master is at 1.1.0.BUILD-SNAPSHOT, no milestone releases and no work to keep up with the 1.1.x versions of the deployer yet.
